### ISSUE TITLE
Not working on linux

### ISSUE THREAD

**Issue by UltiRequiem** [release-lab/install#1]
Example with dvm

https://github.com/axetroy/dvm

https://user-images.githubusercontent.com/71897736/158864221-eaad9e62-4080-4030-84cb-39a84c470596.mp4

Same on bash

![image](https://user-images.githubusercontent.com/71897736/158864860-d9b6312f-5747-4c9b-bed9-21499af7d3a9.png)

**Comment by atjn** [comment]
Also see https://github.com/axetroy/dvm/issues/98

**Comment by atjn** [comment]
You can fix this by putting `sudo` in the the part of the command that requires admin privileges:
```bash
curl -fsSL https://github.com/release-lab/install/raw/v1/install.sh | sudo bash -s -- -r=axetroy/dvm
```

**Comment by axetroy** [comment]
closed by #3
